ada/language/ast.py

```diff
@@ -1008,13 +1008,49 @@ def child_unit(name_expr, scope_expr, dest_env=None,
     )
 
 
+class DocAnnotation(Struct):
+    """
+    Documentation annotation.
+    """
+    key = UserField(T.String)
+    val = UserField(T.String)
+
+
 @abstract
 class BasicDecl(AdaNode):
     """
     Root class for an Ada declaration (RM 3.1). A declaration associates a name
     with a language entity, for example a type or a variable.
     """
 
+    @langkit_property(public=True, external=True,
+                      return_type=DocAnnotation.array,
+                      uses_entity_info=False, uses_envs=False)
+    def get_doc_annotations():
```

Comment:
For the public API, I would suggest to get rid of the `get_` prefix for functions, looks redundant (all functions return something, so they could all start with `get_`).

Also what about getting rid of the doc/documentation inconsistency with the other added property? All other names seem to use the abbreviated, so let’s use only this?